static/css/tools/stat_var_widget.scss

```diff
@@ -149,7 +168,7 @@ $transparent-red: rgba(240, 230, 231, 0.2);
 }
 
 .node-title {
-  min-width: $widget-min-width - 2 * $container-outer-horizontal-margin;
+  min-width: $widget-min-width - 6 * $container-outer-horizontal-margin;
```

Comment:
Out of curiosity, is there a particular reason why the change from `2` to `6`?

Reply:
Made this change to get rid of a horizontal scrollbar, but you're right it's not clear. I reverted the change back to 6, and instead subtracted the width of the resize handle. Also added a comment